src/patients/new/NewPatientForm.tsx

```diff
@@ -19,6 +19,7 @@ const NewPatientForm = (props: Props) => {
   const [isEditable] = useState(true)
   const { onCancel, onSave } = props
   const [approximateAge, setApproximateAge] = useState(0)
+  const [errorMessage, setError] = useState('')
```

Comment:
Rename `setError` to `setErrorMesssage` to keep the state variable and state update function consistent.